2015/proposals/agarwal-darshan.md

```diff
@@ -0,0 +1,133 @@
+Enhance Amy, a Django application for managing workshops
+
+## Abstract
+Amy is a web-based application built using Django framework, for management and administration of workshops. The number of workshops run by Software Carpentry are increasing and currently the organization of workshops is done by volunteers in bits and pieces. So, Amy was started to simplify workshop management. The main focus is to make it easy for administrators to create workshops, keep track of what workshops are being arranged, who are the instructors and what they can teach, who are the learners and so on.
+Currently, basic functionality for Amy is in place. 
+
+This project deals with enhancing Amy by adding many new features to it and make it use a lot more testing.
+The project consists of two main sections:
+	1) Thorough unit testing of existing application and finding out bugs if any and thereby fixing it.
+	2) Adding following features for easy management of events:
+		-Integration with Github for authentication.
+		-Integration with Eventbrite to check :
+			-If the registration of the event exists
+			-If the registration of the event is up and running 
+	3)Integration with mailman
+		-Create a mailing lists for each workshop with instructors, helpers and hosts on it.
+		-Displaying which lists are person signed up to.
+		-Displaying members signed up to a list.
+		-Displaying lists per workshop.
+		-Allowing hosts to co-manage the mailing lists for the workshops along with admins. 
+	4)Reports (with graphs visualisation):
+		-Number of workshops held in the last month/year/day.
+		-Number of people taught in the last month/year/day.
+		-Number of instructors who have taught in the last month/year/day.
+		-Number/List of instructors who have never taught in the last month/year/day.
+		-Number/List of instructors who are about to teach in the last month/year/day.
+		-Event with most number of people/instructors in the last month/year/day.
```

Comment:
“… with the most number of people/instructors…” → “… with the most people/instructors…”